### PR TITLE
Load-based splitting

### PR DESCRIPTION
**Pull request by sploiselle**
@nvanbenschoten Could I get you to do a tech edit of this page? And what is Ridwan's GitHub handle so I can grab his review, as well?

### CODE REVIEW

**Comment by sploiselle** [comment]
@rmloveland Could I get a copy edit _sils vous plait_?

File: v2.2/load-based-splitting.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Some workloads might find splitting ranges more aggressively (i.e. a lower QPS threshold) can improve performance.
+
+On the other hand, some workloads with very large machines might want to increase the QPS threshold to split more conservatively.
+
+## How load-based splitting works
+
+Whenever a range exceeds the cluster's setting for `kv.range_split.load_qps_threshold`, the range becomes eligible for load-based splitting.
+
+At that point, we begin gathering per-key metrics to determine whether or not a split would benefit the cluster's performance based on the following metrics:
```
Comment (rmloveland) [inline v2.2/load-based-splitting.md]: > based on the following metrics
Further down the page you refer to these as " the balance factor and split crossing heuristics".  I think we should call them heuristics here as well.

File: v2.2/load-based-splitting.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+For example, if the operations on a single key exceed the `kv.range_split.load_qps_threshold` value, it's a good candidate to split the range at that point.
+
+Another example is that if the range has equal access among all keys, whose total operation exceeds `kv.range_split.load_qps_threshold`. By splitting the range at the first set of keys whose total operations exceed the threshold, we can reduce the load on the node. If the "other" range still exceeds the threshold, it will eventually be split again.
+
+In both of these examples, the split would only occur if the balance factor and split crossing heuristics determined the split would produce better results.
+
+## Why load-based splitting works
+
+CockroachDB creates a relatively even distribution of leaseholders throughout your cluster (leaseholders are a single replica of a range that both serve reads and coordinate writes operations; [learn more about leaseholders](architecture/replication-layer.html#leases)).
```
Comment (rmloveland) [inline v2.2/load-based-splitting.md]: > learn more about leaseholders

Suggest rewording to something more boring like

> For more information about leaseholders, see the Replication Layer documentation.

File: v2.2/load-based-splitting.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+However, without load-based splitting this distribution is unaware of the load present on any set of keys. This means that even with an equitable distribution of leases throughout the cluster, some leases will generate more traffic for the node that houses them than others. Because each node can only provide so much throughput, a single node can become a bottleneck for providing access to a subset of data in your cluster.
```
Comment (rmloveland) [inline v2.2/load-based-splitting.md]: > without load-based splitting this distribution is unaware of the load present on any set of keys

AIUI the cluster is what has the awareness, not the distribution.  #nitpicking

Suggest rewording to something like

> without load-based splitting, the cluster is unaware of the load present on any set of keys in this distribution

... assuming that a sentence like mine above does not introduce some subtle technical error, of course.

File: v2.2/load-based-splitting.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+However, by leveraging load-based splitting, the cluster can understand load on a per-range basis and split up ranges that generate a significant amount of load into multiple ranges, and therefor multiple leaseholders. This lets the cluster express its load as a function of leases; so the roughly equal distribution of leases also generates a roughly equal distribution of traffic, preventing individual nodes from becoming bottlenecks for your cluster.
```
Comment (rmloveland) [inline v2.2/load-based-splitting.md]: typo: `s/therefor/therefore/`
